src/reporting/private/dashboard.ts

```diff
@@ -86,6 +86,37 @@ export const dataSetActions = [
   'quicksight:CancelIngestion',
 ];
 
+export const analysisPermissionActions = [
+  'quicksight:DescribeAnalysis',
+  'quicksight:UpdateAnalysisPermissions',
+  'quicksight:QueryAnalysis',
+  'quicksight:UpdateAnalysis',
+  'quicksight:RestoreAnalysis',
+  'quicksight:DeleteAnalysis',
+  'quicksight:DescribeAnalysisPermissions',
+];
+
+export const dashboardPermissionActions = [
+  'quicksight:DescribeDashboard',
+  'quicksight:ListDashboardVersions',
+  'quicksight:UpdateDashboardPermissions',
+  'quicksight:QueryDashboard',
+  'quicksight:UpdateDashboard',
+  'quicksight:DeleteDashboard',
+  'quicksight:DescribeDashboardPermissions',
+  'quicksight:UpdateDashboardPublishedVersion',
+];
+
+export const CLICKSTREAM_RETENTION_VIEW_PLACEHOLDER = 'clickstream_retention_view';
+export const CLICKSTREAM_SESSION_VIEW_PLACEHOLDER = 'clickstream_session_view';
+export const CLICKSTREAM_USER_DIM_VIEW_PLACEHOLDER = 'clickstream_user_dim_view';
+export const CLICKSTREAM_EVENT_VIEW_PLACEHOLDER = 'clickstream_event_view';
+export const CLICKSTREAM_DEVICE_VIEW_PLACEHOLDER = 'clickstream_device_view';
+export const CLICKSTREAM_EVENT_PARAMETER_VIEW_PLACEHOLDER = 'clickstream_event_parameter_view';
+export const CLICKSTREAM_LIFECYCLE_DAILY_VIEW_PLACEHOLDER = 'clickstream_lifecycle_daily_view';
+export const CLICKSTREAM_LIFECYCLE_WEEKLY_VIEW_PLACEHOLDER = 'clickstream_lifecycle_weekly_view';
+export const CLICKSTREAM_USER_ATTR_VIEW_PLACEHOLDER = 'clickstream_user_attr_view';
```

Comment:
Why introduce the duplicated and unused constnat again?

https://github.com/awslabs/clickstream-analytics-on-aws/blob/c67d3c1956d89c39351d4f2cecc4e55ef06a9ad6/src/common/constant.ts#L345